client-server-script

```diff
@@ -27,7 +27,7 @@ function do_roadblock() {
     local cmd=""
     cmd="$cmd /usr/bin/roadblock.py --role=follower --redis-server=$roadblock_server"
     cmd="$cmd --timeout=$timeout --redis-password=$roadblock_passwd"
-    cmd="$cmd --uuid=$roadblock_id:$label --follower-id=$cs_id $extra"
+    cmd="$cmd --uuid=$roadblock_id:$label --follower-id=$cs_label $extra"
```

Comment:
I don't care one way or the other, but I'm just curious what the reasoning behind this change in variable name was (especially since it's so pervasive in this PR).  Is this indicative of some issue you were trying to resolve?

Reply:
I am just trying to avoid confusion as to what cs_id means vs cs_label.  An "id" is  just the number while label is the $type-$id, like client-1.  When first writing this I did not need to use just the id (1, 2..) but now I do in some places.

Reply:
ack